list.go

```diff
@@ -10,12 +10,25 @@ import (
 	"text/tabwriter"
 	"time"
 
+	"encoding/json"
+
 	"github.com/Sirupsen/logrus"
 	"github.com/codegangsta/cli"
 	"github.com/opencontainers/runc/libcontainer"
 )
 
-const formatOptions = `table, json, yaml, xml, or "go=<go-template text>"`
+const formatOptions = `table or json`
+
+type containerList struct {
+	ContainerStates []*containerState `json:"containerlist"`
```

Comment:
also you don't need this containerList type at all, just return the `[]containerState` list and marashal it so the output is `[{},{},{}]` instead of something like

```json
{
    "containers": [{},{},{}]
}
```